src/garage/torch/misc/tensor_utils.py

```diff
@@ -0,0 +1,15 @@
+"""utility functions for PyTorch tensors."""
+import torch
+import torch.nn.functional as F  # noqa
```

Comment:
which QA rule is this violating? perhaps we make add a blanket exception to the configs.

please only add exceptions for specific rules. it helps us track which rules are creating problems, and also allows us to quickly remove exceptions when we find a solution.

Reply:
removed